### DISCUSSION TITLE
Macro Break

### DISCUSSION THREAD

**Discussion by BioFUSION3k** [WarmUpTill/SceneSwitcher#903] [Q&A]
I'm just curious, is their a way to break a macros run? I find myself testing a complex intro, and their are some 2-3min wait times, but if something goes wrong I tend to just have to wait for it to end.... Is there a way to break the run so I can change the values again and retry without waiting 10 mins? (buncha different changes during a 10 minute intro wait so it doesnt seem so stale)

Sorry, had to ask this here, Fenrir didnt like me cursing so much, I apologized that my father was a "ghost hunter on youtube"...(on account not one can go 2 secs without cursing) Seemed to hit a nerve, and he banned me from the OBS forum.

**Comment by WarmUpTill** [comment]
The easiest way is probably to simply stop and re-start the scene switcher on the General tab / the dock.
Alternatively you can use the action type "Macro" which will allow you to stop currently running macros, even if they are marked to be running "in parallel".

Let me know if run into any issues! :)

And sorry about the delayed reply!
Have been knocked out with a cold the last few days.
